src/model/model.rs

```diff
@@ -355,7 +370,18 @@ impl Matcher for AddressPattern {
                 let bmask = !0u128 << (128 - prefix);
                 u128::from(*addrp) & bmask == u128::from(*addr) & bmask
             }
-            (P::Domain { pattern }, Address::Domain(domain, _)) => pattern.is_match(domain),
+            (P::Domain(DomainPattern::Regex { pattern: reg }), Address::Domain(domain, _)) => {
+                reg.is_match(domain)
+            }
+            (P::Domain(DomainPattern::Wildcard { wildcard: s }), Address::Domain(domain, _)) => {
```

Comment:
same as above. (or change the `s` to a more readable name.)

Reply:
done with b2baa50